index.js

```diff
@@ -2,60 +2,60 @@ const readline = require("readline");
 const rl = readline.createInterface(process.stdin, process.stdout);
 
 function ask(questionText) {
-	return new Promise((resolve, reject) => {
-		rl.question(questionText, resolve);
-	});
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
 }
 
 start();
 
 async function start() {
-	let lowerBound = 0;
-	let higherBound = 100;
-	let guess = average(higherBound, lowerBound);
-	console.log(
-		"Let's play a game where you (human) make up a number and I (computer) try to guess it."
-	);
-	let yesOrNo = await ask("Is it " + guess + "? ");
-	let yesOrNoFinal = yesOrNo.toLowerCase();
+  let lowerBound = 0;
+  let higherBound = 100;
```

Comment:
You can have the user set the higherBound either by assiigning it to `process.argv[2] || someDefaultValue` or having an `await ask(...)`  which reassigns the higherBound value

Reply:
I thought I already did this. I'm not sure why it isn't in there. I must have forgotten to sent it to github.